packages/block-editor/src/components/rich-text/index.js

```diff
@@ -493,9 +493,10 @@ export class RichText extends Component {
 			const newColor = computedStyle.color
 				.replace( ')', ', 0.2)' )
 				.replace( 'rgb', 'rgba' );
+			const path = `[contenteditable]:focus ${ boundarySelector }`;
+			const rules = `{background-color: ${ newColor }}`;
 
-			globalStyle.innerHTML =
-				`*:focus ${ boundarySelector }{background-color: ${ newColor }}`;
+			globalStyle.innerHTML = path + rules;
```

Comment:
The composing of this string is a bit hard to follow for me, likely a consequence of the placement of `{` (not helped by template strings also having the `{`). Is the purpose to have shorter lines? I'd be more okay with the longer line, personally. There's also an opportunity for an early return for the negated condition, to de-indent by one.

```js
if ( ! element ) {
	return;
}
```